galaxy/api/access.py

```diff
@@ -330,6 +330,20 @@ def can_read(self, obj):
         return True
 
 
+class CommunitySurveyAccess(BaseAccess):
+    def can_read(self, obj):
+        return True
+
+    def can_add(self, data):
+        return self.user.is_authenticated()
+
+    def can_change(self, obj, data):
+        return self.user.is_authenticated()
```

Comment:
Should I only be allowed to modify a survey that I created? If so, think we can add that check here.